app.rb

```diff
@@ -0,0 +1,30 @@
+require 'sinatra/base'
+require 'pg'
+
```

Comment:
To make this run using rackup, I had to make a few adjustments. First I required both the files in the model in the controller using 
`require './lib/peeps.rb'`
`require './lib/user.rb'`